src/pyfme/environment/isa.py

```diff
@@ -0,0 +1,132 @@
+# coding: utf-8
+"""ISA functions.
+"""
+
+from math import exp
+
+# Constants
+R_a = 287.05287  # J/(Kg·K)
+g0 = 9.80665  # m/s^2
+
+# From: https://en.wikipedia.org/wiki/U.S._Standard_Atmosphere
```

Comment:
It is in the `atm` function in line 54 (references). Should we also include the references in the module docstring?